logic/src/commonMain/kotlin/com/wire/kalium/logic/data/service/ServiceRepository.kt

```diff
@@ -58,4 +89,11 @@ internal class ServiceDataSource internal constructor(
             id = serviceMapper.fromModelToDao(serviceId = serviceId),
             conversationId = conversationId.toDao()
         ).map { it?.toModel() }
+
+    private suspend fun getAllServicesAndMapFlow(): Flow<List<ServiceDetails>> =
+        serviceDAO.getAllServices().map {
+            it.map { serviceEntity ->
+                serviceMapper.fromDaoToModel(service = serviceEntity)
+            }
+        }
```

Comment:
```suggestion
    private suspend fun getAllServicesAndMapFlow(): Flow<Either<StorageFailure, List<ServiceDetails>>> =
        serviceDAO.getAllServices().map {
            it.map { serviceEntity ->
                serviceMapper.fromDaoToModel(service = serviceEntity)
            }
        }.wrapStorageRequest()
```

can we handle errors in this case